### ISSUE TITLE
*.java failed to format with Eclipse Code Formatter. Invalid byte 1 of 1-byte UTF-8 sequence.

### ISSUE THREAD

**Issue by Jacksonary** [krasa/EclipseCodeFormatter#216]
My IDEA is IntelliJ 2020.2.1，EclipseCodeFormatter's version is 20.3.201.6668.0-Eclipse_2020-06-JavaOnly, but when i format  my code, always occur error like this: 

```log
*.java failed to format with Eclipse Code Formatter. Invalid byte 1 of 1-byte UTF-8 sequence.
```

Thanks, please give me some help.

**Comment by krasa** [comment]
anything in the log?

**Comment by krasa** [comment]
Fixed by #215, I only forgot to release it :)

**Comment by Jacksonary** [comment]
@krasa Thinks, log just like you saw as previous:

```log
*.java failed to format with Eclipse Code Formatter. Invalid byte 1 of 1-byte UTF-8 sequence.
```

 I have deal with it, just set my `codestyle.xml` as GBK, default is `UTF-8`,it confuse me, because I can format my code previous.
